w08h03/test/pgdp/datastructures/RandomTest.java

```diff
@@ -0,0 +1,121 @@
+package pgdp.datastructures;
+
+import org.junit.jupiter.api.RepeatedTest;
+
+import java.nio.charset.StandardCharsets;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Random;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RandomTest {
+    Random intTest = new Random(100);
+    Random doubleTest = new Random(100);
+    Random dateTest = new Random(100);
+    Random stringTest = new Random(100);
+    
+    final int numberOfTests = 1000;
+    // If you want to increase the number of runs per test, simply increase 
+    // this number.
+    //
+    // Please note that the tests might be slow, if you are using VS Code.
+    // IntelliJ seems to be much faster.

```

Comment:
```suggestion
    // If you want to increase the number of runs per test, simply increase 
    // this number.
    //
    // Please note that the tests might be slow, if you are using VS Code.
    // IntelliJ seems to be much faster.
    final int numberOfTests = 1000;
```

Move comment above the variable.